jumanji/environments/combinatorial/routing/rb_board_generator_v1.py

```diff
@@ -0,0 +1,319 @@
+#from constants import EMPTY, OBSTACLE, SOURCE, TARGET, HEAD, NOOP, LEFT, LEFT, UP, RIGHT, DOWN
+from constants import SOURCE, TARGET, HEAD
+from dataclasses import dataclass
+import numpy as np
+from copy import deepcopy
+import random
+WIRE = SOURCE
```

Comment:
global variables can cause issues so python avoids using them. The below overcomes the naming issue while not adding a new global variable
```suggestion
from constants import SOURCE as WIRE
```